apps/studio/components/interfaces/Support/SupportFormV2.tsx

```diff
@@ -324,12 +337,15 @@ export const SupportFormV2 = ({ setSentCategory, setSelectedProject }: SupportFo
                     <SelectValue_Shadcn_ asChild placeholder="Select an organization">
                       <div className="flex items-center gap-x-2">
                         {(organizations ?? []).find((o) => o.slug === field.value)?.name}
-                        {isLoadingSubscription && <Loader2 size={14} className="animate-spin" />}
+                        {isLoadingSubscription && organizationSlug !== 'no-org' && (
+                          <Loader2 size={14} className="animate-spin" />
+                        )}
                         {isSuccessSubscription && (
                           <Badge variant="outline" className="capitalize">
                             {subscriptionPlanId}
                           </Badge>
                         )}
+                        {organizationSlug === 'no-org' && <span>No specific org</span>}
```

Comment:
let's spell this out fully, same for below
```suggestion
                        {organizationSlug === 'no-org' && <span>No specific organization</span>}
```